app/views/spree/layouts/spree_application.html.erb

```diff
@@ -4,25 +4,24 @@
 <!--[if IE 8 ]>    <html class="ie ie8" lang="<%= I18n.locale %>"> <![endif]-->
 <!--[if IE 9 ]>    <html class="ie ie9" lang="<%= I18n.locale %>"> <![endif]-->
 <!--[if gt IE 9]><!--><html lang="<%= I18n.locale %>"><!--<![endif]-->
-  <head data-hook="inside_head">
-    <%= render partial: 'spree/shared/head' %>
-  </head>
-  <body class="<%= body_class %>" id="<%= @body_id || 'default' %>" data-hook="body">
-    <%= render partial: 'spree/shared/header' %>
-
-    <div class="container">
-      <div class="row" data-hook>
-        <%= spree_breadcrumbs(@taxon) %>
-
-        <%= render partial: 'spree/shared/sidebar' if content_for? :sidebar %>
-
-        <div id="content" class="<%= !content_for?(:sidebar) ? "col-sm-12" : "col-sm-8 col-md-9" %>" data-hook>
-          <%= flash_messages %>
-          <%= yield %>
-        </div>
-
-        <%= yield :templates %>
+<head data-hook="inside_head">
+  <%= render partial: 'spree/shared/head' %>
+</head>
+<body  id="<%= @body_id || 'default' %>" data-hook="body">
+<div id="wrapper">
+   <%= render partial: 'spree/shared/header' %>
+   <div>
+      <div  data-hook>
+         <%= spree_breadcrumbs(@taxon) %>
+         <%= render partial: 'spree/shared/sidebar' if content_for? :sidebar %>
```

Comment:
indentation issues divs are not properly closed see the body tag is not closed correctly